api/controllers/webhook.js

```diff
@@ -44,6 +50,37 @@ const findSiteForWebhookRequest = (request) => {
     });
 };
 
+const organizationWebhookRequest = async (payload) => {
+  const {
+    action, membership, organization,
+  } = payload;
+  const { login: orgName } = organization;
+  if (orgName !== config.federalistUsers.orgName) {
+    logger.warn(`Not a ${config.federalistUsers.orgName} membership action:\t${JSON.stringify(payload)}`);
+    return;
+  }
+
+  const { user: { login } } = membership;
+  const username = login.toLowerCase();
+  const user = await User.findOne({ where: { username } });
+
+  if (['member_added', 'member_removed', 'member_invited'].includes(action)) {
+    EventCreator.audit(Event.labels.FEDERALIST_USERS, user || User.build({ username }), payload);
```

Comment:
hmmm, do we care about this if there is no user? also `User.build` will only build a user in memory so there won't be an `id`...

Reply:
the EventCreator.audit expects a model.  This is User related data ... it is fine that it has no id

Reply:
The EvenCreator.audit has error handling but i can move it below